Source/santametricservice/Formats/SNTMetricRawJSONFormat.m

```diff
@@ -0,0 +1,100 @@
+/// Copyright 2021 Google Inc. All rights reserved.
+///
+/// Licensed under the Apache License, Version 2.0 (the "License");
+/// you may not use this file except in compliance with the License.
+/// You may obtain a copy of the License at
+///
+///    http://www.apache.org/licenses/LICENSE-2.0
+///
+///    Unless required by applicable law or agreed to in writing, software
+///    distributed under the License is distributed on an "AS IS" BASIS,
+///    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+///    See the License for the specific language governing permissions and
+///    limitations under the License.
+#import "Source/common/SNTLogging.h"
+
+#import "Source/santametricservice/Formats/SNTMetricRawJSONFormat.h"
+
+@implementation SNTMetricRawJSONFormat {
+  NSDateFormatter *_dateFormatter;
+}
+
+- (instancetype)init {
+  self = [super init];
+  if (self) {
+    _dateFormatter = [[NSDateFormatter alloc] init];
+    [_dateFormatter setDateFormat:@"yyyy-MM-dd'T'HH:mm:ss.SSS'Z'"];
+  }
+  return self;
+}
+
+- (NSArray *)normalizeArray:(NSArray *)arr {
+  NSMutableArray *normalized = [NSMutableArray arrayWithArray:arr];
+
+  for (int i = 0; i < [arr count]; i++) {
```

Comment:
`for (id obj in arr) {`

Reply:
In this case I want the array index so I can replace the item at that location.

Reply:
I switched to Block enumeration.